src/document/node.js

```diff
@@ -232,19 +232,14 @@ CKEDITOR.define( [ 'document/attribute', 'utils', 'ckeditorerror' ], ( Attribute
 		}
 
 		/**
-		 * Gets the number of attributes.
+		 * Returns attribute iterator. It can be use to create a new element with the same attributes:
 		 *
-		 * @protected
-		 * @returns {Number} Number of attributes.
+		 *	  const copy = new Element( element.name, element.getAttrIterator() );
+		 *
+		 * @returns {Iterable.<document.Attribute>} Attribute iterator.
 		 */
-		_getAttrCount() {
-			let count = 0;
-
-			for ( let attr of this._attrs ) { // jshint ignore:line
-				count++;
-			}
-
-			return count;
+		getAttrIterator() {
```

Comment:
Actually, after a sec I'm OK with the iterator. But the method should definitely not state that. It should be `getAttrs()`. There simply needs to be one method of accessing attributes and that's what this method will do.